Makefile

```diff
@@ -182,6 +182,18 @@ TOUCHID_MESSAGE := with Touch ID
 TOUCHID_TAG := touchid
 endif
 
+# Build teleport/api against libsclite?
+# Only build if LIBPCSCLITE=yes, or if it's otherwise available.
+# This is used for PIV functionality.
+LIBPCSCLITE_MESSAGE := without libpcsclite
+ifeq ("$(LIBPCSCLITE)", "yes")
+LIBPCSCLITE_MESSAGE := with libpcsclite
+LIBPCSCLITE_BUILD_TAG := libpcsclite
+else ifeq ("$(shell pkg-config libpcsclite 2>/dev/null; echo $$?)", "0")
```

Comment:
Can we add a way to opt-out of this? We don't do any autodetection for `libfido`, and there's at least ways to force the Rust builds and libbpf off.